x-team/xp-ui
# Rename "Closed Jobs" to "Previous Applications"

**Release Type**: UI Improvement

Fixes https://x-team-internal.atlassian.net/browse/XP-3474

## Description

* Change the user-visible wording "Closed Applications" to "Previous Applications".
* Update component names from `ClosedJobApplications` to `PreviousJobApplications`
* Rename component files to align with component names
* Update other relevant variable names from `closed` to `previous`

## Checklist


## Steps to Test or Reproduce

Start the storybook app in XP-UI, then visit http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout-use-cases--my-applications-page. Scroll down to the bottom to check out the new wording.

## Impacted Areas in Application

JobsPageLayout, the "My Applications" area
